org.keshavnrj.wonderwall.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: org.keshavnrj.wonderwall
+base: io.qt.qtwebkit.BaseApp
+base-version: '5.11'
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: wonderwall
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
+  - --filesystem=xdg-download
+  - --device=dri
+  
+  - --talk-name=org.gtk.vfs
```

Comment:
```suggestion
```

This one has always been a mistake

Reply:
Will remove this in next PR, I found this line in other flathub flatpaks too :(